versions.properties

```diff
@@ -3,32 +3,40 @@
 # via scala-dist-pom.xml and scala-library-all-pom.xml
 # when adding new properties that influence a release,
 # also add them to the update.versions mechanism in build.xml,
-# which is used by scala-release-2.11.x in scala/jenkins-scripts
+# which is used by the release script scripts/jobs/integrate/bootstrap
+
+# The scala version used for boostrapping. This has no impact on the final classfiles:
+# there are two stages (locker and quick), so compiler and library are always built
+# with themselves. Stability is ensured by building a third stage (strap).
 starr.version=2.11.6
-starr.use.released=1
 
 # These are the versions of the modules that go with this release.
 # These properties are used during PR validation and in dbuild builds.
 
-# e.g. 2.11.0-RC1, 2.11
+# The scala.binary.version determines how modules are resolved. For example, it
+# determines which partest artifact is being used for running the tests.
+# It has to be set in the following way:
+#  - After 2.x.0 is released, the binary version is 2.x.
+#  - During milestones and RCs, modules are cross-built against the full version.
+#    So the value is the full version (e.g. 2.12.0-M1).
 scala.binary.version=2.11
 # e.g. 2.11.0-RC1, 2.11.0, 2.11.1-RC1, 2.11.1
 # this defines the dependency on scala-continuations-plugin in scala-dist's pom
 scala.full.version=2.11.6
 
 # external modules shipped with distribution, as specified by scala-library-all's pom
-scala-xml.version.number=1.0.3
-scala-parser-combinators.version.number=1.0.3
+scala-xml.version.number=1.0.4
+scala-parser-combinators.version.number=1.0.4
 scala-continuations-plugin.version.number=1.0.2
 scala-continuations-library.version.number=1.0.2
-scala-swing.version.number=1.0.1
-akka-actor.version.number=2.3.4
```

Comment:
I wasn't sure what our policy was here (upgrade akka or not?) I checked the history of this file and found that we have bumped the version twice in the 2.11.x series: ef427b27a / 103c9850c, so I think this is fine.